src/middleware/auth.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
```

Comment:
Please use the auth.middleware.js instead of creating a new one.

Reply:
noted